<!-- Dynamixel Motors launch file -->

<launch>
  #### Motors ###########################################################

  <node pkg="motors" type="motor_monitor" name="motor_monitor"
    output="screen">
  </node>


</launch>
